WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -171,20 +177,22 @@ private extension BlazeCampaignDashboardView {
         Button {
             createCampaignTapped?(nil)
         } label: {
-            Text(Localization.promote)
-                .foregroundColor(.init(uiColor: .accent))
+            Text(Localization.createCampaign)
+                .fontWeight(.bold)
```

Comment:
Nit: bold seems a bit too much - I think we should use semi-bold instead or keep the default style of the secondary button style to be consistent with other buttons.

Reply:
Good point. +1 for keeping the button consistent for now. https://github.com/woocommerce/woocommerce-ios/pull/12593/commits/abf53be74e7e0435335db506c2d19ba381887ac2